app/src/main/java/io/getstream/chat/android/core/poc/library/requests/QueryUsers.kt

```diff
@@ -0,0 +1,56 @@
+package io.getstream.chat.android.core.poc.library.requests
+
+import com.google.gson.annotations.Expose
+import com.google.gson.annotations.SerializedName
+import io.getstream.chat.android.core.poc.library.FilterObject
+
+data class QueryUsers(
+
+    @SerializedName("filter_conditions")
+    @Expose
+    private var filter: FilterObject? = null,
+
+    @SerializedName("sort")
+    @Expose
+    private val sort: QuerySort? = null,
+
+    @SerializedName("presence")
```

Comment:
There is no need to repeat field name with `@SerializedName` if it's already correct.